go-chaos/release.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if ! command -v gh &> /dev/null
+then
+    echo "gh (github cli) could not be found"
+    echo "Please make sure to install it before."
+    echo "Check: https://github.com/cli/cli/blob/trunk/docs/install_linux.md"
+    exit 1
+fi
+
+
+echo "Nice, you want to release a new version of zbchaos."
+echo "Checking out main and retrieving most recent tags"
+git checkout main
+git fetch --all
+git pull origin main
+
+lastRelease=$(git tag | sort -Vr | head -n1)
+
+echo "Last release was: $lastRelease"
+read -p "Please type a new release here: " newRelease
```

Comment:
:x: This won't work when run from GH actions. Maybe allow providing `newRelease` as an environment variable?

Reply:
Thanks good catch, totally forgot about this 😆

Reply:
Thanks for checking @oleschoenburg should be fixed